---
string[] names
float32[] values
Error status
